lib/telegraf/railtie.rb

```diff
@@ -48,48 +48,59 @@ class Railtie < ::Rails::Railtie
     # Connect URI or tuple
     config.telegraf.connect = ::Telegraf::Agent::DEFAULT_CONNECTION
     config.telegraf.tags = {}
+    config.telegraf.before_send = nil
 
     # Install Rack middlewares
     config.telegraf.rack = ::ActiveSupport::OrderedOptions.new
     config.telegraf.rack.enabled = true
     config.telegraf.rack.series = 'requests'
     config.telegraf.rack.tags = {}
+    config.telegraf.rack.before_send = nil
 
     # Install request instrumentation
     config.telegraf.instrumentation = true
 
     # Install Grape instrumentation
     config.telegraf.grape = ::ActiveSupport::OrderedOptions.new
     config.telegraf.grape.enabled = defined?(::Grape)
+    config.telegraf.grape.before_send = nil
```

Comment:
This one doesn't seem to be passed on. 🧐

Reply:
Correct. `grape` only adds data to the point in the rack env.

Will remove that one.